src/components/tests/mocks/modifiedCharacters.ts

```diff
@@ -0,0 +1,89 @@
+import { ModifiedCharacter } from '../../../api/starWars-types'
+export const modifiedCharacters: ModifiedCharacter[] = [
```

Comment:
[nitpick] - you wanna name this variable something that tells that it is not real data. I would name it something like the followings;

- `mockedModifiedCharacters`
- `modifiedCharactersFixture`